test_tableslurp.sh

```diff
@@ -50,32 +55,32 @@ export AWS_SECRET_ACCESS_KEY="$AWS_SEC_ACC_KEY"
 # part into index file as content & copy the 2 files up to S3. Example files:
 # /tmp/tablesnap-test/test_aws_s3_cp.txt
 # /tmp/tablesnap-test/index.txt
-TEST_FILE="test_aws_s3_cp.txt"
+UPDOWN_FILE="aws_s3_cp.txt"
 INDEX_FILE="index.txt"
-RENAMED_FILE="test_file_orig.txt"
+ORIG_TEST_FILE="orig_test_file.txt"
 CONTAINER="tester"
-SRC_LOCAL="${DIR_LOCAL}/${TEST_FILE}"
+SRC_LOCAL="${DIR_LOCAL}/${UPDOWN_FILE}"
 INDEX_SRC="${DIR_LOCAL}/${INDEX_FILE}"
 INDEX_BUCKET="${BUCKET}"
 BACKUP_BUCKET="${BUCKET}"
+ORIG_TEST_FILE_LOC="${DIR_LOCAL}"/"${ORIG_TEST_FILE}"
 
 # Create destinations on S3. Note: Dest for index file is at top-level,
 # i.e., not under a directory.
 # s3://kimlew-tablesnap-tests/tester:/tmp/tablesnap-test/test_aws_s3_cp.txt
-DEST_ON_S3=s3://"${BUCKET}"/"${CONTAINER}":"${DIR_LOCAL}"/"${TEST_FILE}"
+DEST_ON_S3=s3://"${BUCKET}"/"${CONTAINER}":"${SRC_LOCAL}"
 INDEX_DEST=s3://"${BUCKET}"/"${CONTAINER}":/"${INDEX_FILE}"
 
+
 echo "SRC_LOCAL is:" "${SRC_LOCAL}"
 echo "DEST_ON_S3 is:" "${DEST_ON_S3}"
 echo "INDEX_SRC is:" "${INDEX_SRC}"
 echo "INDEX_DEST is:" "${INDEX_DEST}"
 echo
 
-touch "${SRC_LOCAL}" > /dev/null
-touch "${INDEX_SRC}" > /dev/null
-
-echo "${DIR_LOCAL}/${TEST_FILE}" > "${INDEX_SRC}"
-echo "AFTER touch & > index file, the ls of /tmp/tablesnap-test:"
+date > "${SRC_LOCAL}"
+echo "${SRC_LOCAL}" > "${INDEX_SRC}"
+echo "CHECK /tmp/tablesnap-test with ls after ${SRC_LOCAL} given content & $UPDOWN_FILE > index file:"
```

Comment:
I think you meant to use `${UPDOWN_FILE}` with curly braces, just to be consistent.

Reply:
It's just a debugging statement but I've reworded it so it's more clear.